components/PicklistTable/index.tsx

```diff
@@ -102,21 +114,29 @@ export default function PickListTable({
 							<TableHead className='text-center font-semibold' colSpan={2}>Endgame</TableHead>
 						</TableRow>
 						<TableRow>
-							<TableHead>Number</TableHead>
-							<TableHead>First Pickability</TableHead>
-							<TableHead>Second Pickability</TableHead>
-							<TableHead>Amp</TableHead>
-							<TableHead>Speaker</TableHead>
-							<TableHead>Amp</TableHead>
-							<TableHead>Speaker</TableHead>
-							<TableHead>Amplified Speaker</TableHead>
-							<TableHead>Climb Rate</TableHead>
-							<TableHead>Trap Notes</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary'>Number</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('firstPickability')}>First Pickability</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('secondPickability')}>Second Pickability</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('autoAmpNotes')}>Amp</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('autoSpeakerNotes')}>Speaker</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('teleopAmpNotes')}>Amp</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('teleopSpeakerNotes')}>Speaker</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('teleopAmplifiedSpeakerNotes')}>Amplified Speaker</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('climbRate')}>Climb Rate</TableHead>
+							<TableHead className='hover:bg-accent hover:text-accent-foreground hover:cursor-grab active:cursor-grabbing active:bg-primary' onClick={() => handleHeaderClick('trapNotes')}>Trap Notes</TableHead>
 						</TableRow>
 					</TableHeader>
 					<TableBody>
 						<SortableContext items={items} strategy={verticalListSortingStrategy}>
-							{teams.map((team) => (
+							{teams.filter((team) => team !== undefined).sort((teamA, teamB) => {
```

Comment:
W/drag and drop we might want a custom "sort" order. Right now I think it ignores any change to the DND order.

We might need a conditional where if sortColumn is undefined/null it just maps teams, otherwise it does the sort.

Also in the `handleDragEnd` function set sortColumn to undefined/null